### PR TITLE
Issue #524: Adds 2 column flipped layout

### PR DESCRIPTION
**Pull request by behindthepage**
This solves https://github.com/backdrop/backdrop-issues/issues/524

### CODE REVIEW

File: core/layouts/two_column/layout.css [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /**
  * @file
- * CSS for the three-three-four layout.
```
Comment (behindthepage) [inline core/layouts/two_column/layout.css:3]: I came across this little mistake and fixed it. Should I have created a new issue instead?

Comment (quicksketch) [inline core/layouts/two_column/layout.css:3]: This is perfectly acceptable to include. :+1:

**Comment by behindthepage** [comment]
This has failed with this error
ERROR: FATAL AJAXMultiFormTestCase: test runner returned a non-zero error code (255).

Do I need to do something about this? And if so, what?

Regards
Clueless ;-)

**Comment by quicksketch** [comment]
Hm, that's odd. I'll rerun the tests and see if that helps.

**Comment by quicksketch** [comment]
Yay, passed this time.

I tested this out locally and it works great. Thanks for the excellent consistency with the existing 2-col layout. They're nearly identical (which is the point).

I noticed that your authorship is not set up correctly (your Git commit was recorded as being by "root"), so I amended the commit with the correct name. Committed at https://github.com/backdrop/backdrop/commit/4bd0e82a0298773eb21b960d86cf9d586272a804.

Could you update your Git config your your actual name?

```
git config --global user.name "John Doe"
```

Then future commits will have the right name. Thanks!

**Pull request closed without merging**